src/main/java/com/microsoft/sqlserver/jdbc/SQLServerConnection.java

```diff
@@ -3421,7 +3426,20 @@ int writeFedAuthFeatureRequest(boolean write,
         }
         return totalLen;
     }
+    
+    int writeDataClassificationFeatureRequest (boolean write /* if false just calculates the length */,
```

Comment:
I'd put the protected keyword here explicitly to maintain consistency throughout the file.